proto/agent.proto

```diff
@@ -222,4 +226,41 @@ message AppLaunchConfig {
   string launchUIDescription = 6;
 }
 
+message Onboarding {
+  string type = 1;
+  string projectUrl = 2;
+  string status = 3;
+  bytes  details=4;
+}
+
+message AddOrUpdateOnboardingRequest {
+  string type = 1;
+  string projectUrl = 2;
+  bytes  details=4;
```

Comment:
what is details ?
it need be "accessToken" in string format

type change to gitType

Reply:
gitType change to gitoptsType